src/content/docs/docs/intro/web-monetization-extension.mdx

```diff
@@ -0,0 +1,225 @@
+4. Click **Add a public key**.
+5. Enter a nickname for the key, then paste the key into the public key field.
+6. Click **Save**.
+7. Go back to your extension and try connecting again.
+</Steps>
+
+Contact Interledger Wallet at support@interledger.app if you need additional help.
+
+
+
+
+
+
+
+
+
+
+
+
+## Extension states
+
+When the **Continuous payment** toggle is on, the extension icon appears in color. When the toggle is off, the icon appears in gray. Whether a page is web monetized does not affect the color of the icon.
+
+#### Continuous payment toggled on
+
+| Icon | State |
+| :----: | ----- |
+| <div style="padding-left: 5px;">![Continuous payments turned on in extension, extension is paying](/img/docs/extension/enabled-monetized.png "Active icon state with green circle and checkmark")</div> | Page is web monetized and the extension is continuously paying |
+| <div style="padding-left: 5px;">![Continuous payments turned on in extension, extension is not paying and needs user attention](/img/docs/extension/enabled-attention.png "Active icon state with orange circle and exclamation mark")</div> | Page is web monetized, but the extension needs user attention and is **not** continuously paying |
+| <div style="padding-left: 5px;">![Continuous payments turned on in extension, page is not web monetized](/img/docs/extension/enabled-unmonetized.png "Active icon state with red circle and x")</div> | Page is **not** web monetized, the extension is **not** continuously paying |
+
+#### Continuous payment toggled off
+
+| Icon | State |
+| :----: | ----- |
+| <div style="padding-left: 5px;">![Continuous payments turned off in extension](/img/docs/extension/disabled-monetized.png "Inactive icon state with green circle and checkmark")</div> | Page is web monetized, but the extension is **not** continuously paying |
+| <div style="padding-left: 5px;">![Continuous payments turned off in extension, needs user attention](/img/docs/extension/disabled-attention.png "Inactive icon state with orange circle and exclamation mark")</div> | Page is web monetized, but the extension needs user attention and is **not** continuously paying |
+| <div style="padding-left: 5px;">![Continuous payments turned off in extension, page is not web monetized](/img/docs/extension/disabled-unmonetized.png "Inactive icon state with red circle and x")</div> | Page is **not** web monetized, so the extension is **not** continuously paying |
+
+## FAQ
+
+<details>
+<summary>I get an error from my wallet provider when linking the extension to my wallet. Why?</summary>
+It's hard to say without a specific error message. Two likely scenarios are:
+* You waited too long to accept the connection. Your wallet provider might only give you a short window to accept the connection to keep your account safe. 
+* You accepted the connection too quickly--within seconds. Your wallet provider may do this as a security measure against bots or to ensure you've read the contents of the screen.
```

Comment:
Might be a good advice to mention 5 second wait here (the default in Rafiki, which Interledger Wallet definitely uses).